tests/xtimer_usleep_short/main.c

```diff
@@ -22,17 +22,25 @@
 #define TEST_USLEEP_MIN (0)
 #define TEST_USLEEP_MAX (500)
 
+#define TEST_TIME (125250)
```

Comment:
How did you determine this value ?

Reply:
http://www.wolframalpha.com/input/?i=500%2B499%2B498%2B+...+%2B+1

Reply:
Then threre's direct relationship with TEST_USLEEP_MAX, so I think it's useless.

Reply:
I don't understand what you mean, but changed the value to be calculated.